target_policy:
  alpha: 10.0
  beta: 15
  gamma: 0.001
  factor_A: 10.0

waverider_policy:
  r: 1.3
  c: 0.2
  eta_rep: 22
  nu_rep: 1.4
  eta_damp: 140
  nu_damp: 1.2
  enable_repulsor: true
  enable_damper: true
  occupancy_threshold: 0.01
  run_all_levels: false
  lowest_level_radius: 1.0
  use_only_lowest_level: true
